app/javascript/components/CSVUploadForm/index.js

```diff
@@ -0,0 +1,27 @@
+const SELECTORS = {
+  fileInput: '#csv_upload_form_file'
```

Comment:
```suggestion
  FileInput: '#csv_upload_form_file'
```

![Screen Shot 2021-07-14 at 6 13 55 PM](https://user-images.githubusercontent.com/6076762/125612989-bbd63c1f-42ca-41ba-8058-4d741c0daff8.png)

Reply:
I'm not sure to understand: `SELECTORS` is already screaming_snake_case and for it's key the convention is `lowerCamelCase`.
![image](https://user-images.githubusercontent.com/77609814/125716312-46f0c005-1f0f-4963-986a-e66c8a86be0e.png)
So it seems to me that it's all good here.

Let me know @junan if I missed your point ~

Reply:
My bad! your existing already fine